src/app/(profile)/profile.mappers.ts

```diff
@@ -175,7 +175,8 @@ export const mapSpecializations = (
 
 //HOURLY RATE
 export const hourlyRateOptions: DropdownOption[] = [
-  { name: '< 100 zł/h', value: '0-99' },
+  { name: '< 50 zł/h', value: '10-49' },
```

Comment:
**nit:** Value should be "1-50"

Reply:
Ok, so shoudn't we change other values as well? I mean `50-99`, `100-149` etc

Reply:
I will fix it in the branch with the filter because it will be closed after this branch